scene/gui/rich_text_label.cpp

```diff
@@ -4252,10 +4316,11 @@ void RichTextLabel::append_text(const String &p_bbcode) {
 			pos = brk_end + 1;
 			tag_stack.push_front(tag);
 		} else if (tag.begins_with("table=")) {
-			Vector<String> subtag = tag.substr(6, tag.length()).split(",");
+			int value_pos = tag.find("=");
+			Vector<String> subtag = _split_unquoted(tag.substr(value_pos + 1), U',');
```

Comment:
The `value_pos` and `substr()` in each tag could be replaced with a method that returns tag value, like
```C++
String _get_tag_value(const String &p_tag) const {
	return tag.substr(tag.find("=") + 1);
}
```
(also `find()` could be possibly replaced with `find_char()`)